debian/stretch/foreman/links

```diff
@@ -8,3 +8,5 @@ var/lib/foreman/db		usr/share/foreman/db
 var/log/foreman			usr/share/foreman/log
 var/lib/foreman/public		usr/share/foreman/public
 usr/bin/ruby2.3		usr/bin/foreman-ruby
+usr/sbin/dynflowd usr/share/foreman/script/dynflowd
+usr/sbin/ruby-foreman-tasks usr/share/foreman/script/dynflowd
```

Comment:
this should be a different script, I guess?

Reply:
Why a different script? It's supposed to do the same

Reply:
if I am reading the two lines correctly, they should
1. create a link `usr/share/foreman/script/dynflowd` pointing to `usr/sbin/dynflowd`
2. create a link `usr/share/foreman/script/dynflowd` pointing to `usr/sbin/ruby-foreman-tasks`

and that clashes, as you end up with the same link name

Reply:
I think you're reading it the other way around: it's ```<target> <file>``` since I doubt we're making ```/usr/bin/ruby2.3``` point to ```/usr/bin/foreman-ruby```.

Reply:
That's what I said, it's `<target> <linkname>`, and we define the same linkname for two targets. :)

Reply:
I think you're right that these should be the other way around:
```
usr/share/foreman/script/dynflowd usr/sbin/dynflowd
usr/share/foreman/script/dynflowd usr/sbin/ruby-foreman-tasks
```